### DISCUSSION TITLE
Desktop app for Linux

### DISCUSSION THREAD

**Discussion by Rexogamer** [ananthakumaran/paisa#27] [Q&A]
Heya from HN 😸 

As an Ubuntu user, I'm curious: would it be possible to bring the desktop app to Linux? CLIs are fine but being able to use a GUI would be great. Thanks!

**Comment by avinashkanaujiya** [comment]
you can use gui, you just only need cli, when you are starting app, using `paisa serve`, after that just use the web browser.

**Comment by ananthakumaran** [comment]
I have added build support for the Debian package on master, it will be included in the next release. In the meantime, you can try
[paisa.zip](https://github.com/ananthakumaran/paisa/files/12708373/paisa.zip)
